shared/src/test/scala/squants/experimental/ExperimentalSpec.scala

```diff
@@ -12,36 +12,38 @@ package squants.experimental
 import scala.util.Success
 import org.scalatest.flatspec.AnyFlatSpec
 import org.scalatest.matchers.should.Matchers
+// import squants.experimental.DimensionlessConversions._
+// import squants.experimental.SquantsNumeric._
 
 class ExperimentalSpec extends AnyFlatSpec with Matchers {
 
   behavior of "Experimental Code"
 
-  it should "create an Int based Quantity as previously" in {
```

Comment:
Why are these tests commented out?

Reply:
I had troubles compiling them in dotty, I tried a few variants and gave up, especially considering this is "experimental" code